.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,36 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue related to manifests
+    url: https://github.com/kubeflow/manifests/issues/new
+    about: Create issue in kubeflow/manfiests repo
+  - name: Issue related to deployment
+    url: https://github.com/kubeflow/kfctl/issues/new
+    about: Create issue in kubeflow/kfctl repo
+  - name: Issue related to pipelines
+    url: https://github.com/kubeflow/pipelines/issues/new/choose
+    about: Create issue in kubeflow/pipelines repo
+  - name: Issue related to notebooks
+    url: https://github.com/kubeflow/kubeflow/issues/new/choose
+    about: Creat issue in kubeflow/kubeflow repo
+  - name: Issue related to katib
+    url: https://github.com/kubeflow/katib/issues/new/choose
+    about: Create issue in kubeflow/katib repo
+  - name: Issue related to kfserving
+    url: https://github.com/kubeflow/kfserving/issues/new/choose
+    about: Create issue in kubeflow/kfserving repo
+  - name: Issue related to tf-operator
```

Comment:
I feel like there are too many repos and users who want to file an issue would get lost. Some alternatives (just my $0.02):
1. Categorize the redirect links by working groups.
2. Encourage users to still file issues here but ask them to add labels that are relevant to each working group.

Reply:
> Categorize the redirect links by working groups.

This might be a good idea, what if one WG maintain multiple repos, e.g, training WG, it might doesn't make sense if user has an issue related to pytorch-operator, but sent in tf-operator / common repo.

Reply:
Yeah that would be an issue. Perhaps we can use kubeflow/common to track all training operator related issues. Or follow the second suggestion I had above. BTW, is it possible to do multi-stage selections in the issue template?

Reply:
Yes, multi-stage sounds like a good idea.
Maybe, each WG can decide on a main repo, and add additional redirects using the same approach in the main repo?

I don't like the label approach, because it's hard to configure notification in a proper way.

Reply:
I digged a bit and don't think there's multi-stage in the issue template.  

Not sure what's the best way for a WG who maintains multiple repo, but as of now, currently we only have training-WG maintainning multiple repos, so I think this should be fine without multistage.

Reply:
In fact, I think training WG can maintain a readme.md page somewhere, maybe in GitHub, maybe in kubeflow.org.

The page should describe where issues should be reported. Then here in this repo, we'll just need to provide a link to that page.


For discussion, I think we can go with current PR first if it takes some time to set up that page.